src/extensionPoints/triggerExtension.ts

```diff
@@ -155,6 +159,22 @@ async function interval({
   console.debug("interval:completed");
 }
 
+function pickEventProperties(nativeEvent: Event): UnknownObject {
```

Comment:
This could be typed:

```suggestion
function pickEventProperties(nativeEvent: Event): SerializableKeyboardEvent | null {
```

```ts
interface SerializableKeyboardEvent {
	key: number;
	keyCode: number;
	metaKey: boolean;
	altKey: boolean;
	shiftKey: boolean;
	ctrlKey: boolean;
}
```

Reply:
> This could be typed

I don't think there's much value in typing it yet because the caller doesn't care what it gets back as long as it's serializable